backend/mr-db-sql/src/main/kotlin/io/featurehub/db/services/FeatureSqlApi.kt

```diff
@@ -29,15 +33,28 @@ interface InternalFeatureSqlApi {
 
 class FeatureSqlApi @Inject constructor(
   private val database: Database, private val convertUtils: Conversions, private val cacheSource: CacheSource,
-  private val rolloutStrategyValidator: RolloutStrategyValidator
+  private val rolloutStrategyValidator: RolloutStrategyValidator,
+  private val featureMessagingCloudEventPublisher: FeatureMessagingCloudEventPublisher,
+  executorSupplier: ExecutorSupplier
 ) : FeatureApi, FeatureUpdateBySDKApi, InternalFeatureSqlApi {
+
+  @ConfigKey("messaging.publisher.thread-pool")
+  val threadPoolSize: Int? = 4
+
+  @ConfigKey("messaging.publish.enabled")
+  val messagingPublishEnabled: Boolean? = false
```

Comment:
did you run up the application? and try and change it to true? a val indicates a constant, so it should be a var.

Reply:
I did run the application and all the e2e tests. I have tests that set this configKey to true and all those pass. I am not sure if we need to change this to true/false in runtime though. But I can change this to var in my next ticket if we need to